pkgs/top-level/rust-packages.nix

```diff
@@ -7,9 +7,9 @@
 { runCommand, fetchFromGitHub, git }:
 
 let
-  version = "2017-01-27";
-  rev = "6a73a15e27364a0c191d61d52406bebb7639b657";
-  sha256 = "1dklswbf3acfqid4vv7g2xpqc4gswjgh4ih9xjx3a0m3a69cw9lb";
+  version = "2017-02-13";
+  rev = "1e63b42f967d6bb3338526c3570bf81049153702";
+  sha256 = "17w79fic6ndb724z7zywm1cisha362bjvs8sbvv740nrvsya41gy";
```

Comment:
You can "delete" your last commit by executing `git reset --hard 72c75e795ac86333d9aadcd8ba934bfc06cfe08e` :wink:

I guess you caused that merge commit by running `git pull` in that case you can run `git pull --rebase` (while this PR is open) to avoid that the next time (or use `git fetch` instead but that's a bit more complicated). And for your next PR I would suggest that you create a new branch (i.e. don't use your `master` branch) - this doesn't really matter but normally it will make things easier for you.